assets/large_language_models/rag/environments/rag/context/conda_dependencies.yaml

```diff
@@ -7,6 +7,7 @@ dependencies:
 - pip=21.3.1
 - scikit-learn==0.24.2
 - pip:
+  - azure-ai-formrecognizer
```

Comment:
Thanks for pushing this fix for our customers @jinzhaochangAtMicrosoft !
It's not clear in this PR what this package is for, or how our components use it. Can you please help me understand this change?

Reply:
Project Wednesday will use AML RAG ingestion pipeline to create the index (post Ignite). Our ingestion pipeline will support using Document Intelligence (aka fromrecognizer) to crack the PDF files.

I'm adding the code in a separated PR.

Reply:
Can this PR go with the components change then?
I believe we are currently using pypdf PdfReader - is the goal to switch or duplicate?

Reply:
sorry Jinzhao, I just learned that Wednesday will be use llm-rag-embeddings environment.

Reply:
> Can this PR go with the components change then? I believe we are currently using pypdf PdfReader - is the goal to switch or duplicate?

Okay. We will do that. 
With the new change, we are going to provide an option to use this formrecognizer if the customer passed in the connection name as a new parameter, otherwise we are still using current PDF loader.